src/org/jitsi/util/RTPUtils.java

```diff
@@ -180,6 +181,28 @@ public static int readUint24AsInt(byte[] buffer, int offset)
     }
 
     /**
+     * Returns the given integer masked to 16 bits
+     * @param value the integer to mask
+     * @return the value, masked to only keep the lower
+     * 16 bits
+     */
+    public static int as16Bits(int value)
```

Comment:
wish generics could be used here but they don't work for the built-in types :/ still, i think this reads so much nicer than seeing `value & 0xFFF` in the code that i think it's worth it (though definitely open to a better implementation method if there is one)